dask_sql.Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG CUDA_VER=11.2
+ARG LINUX_VER=ubuntu18.04
+
+FROM gpuci/miniconda-cuda:$CUDA_VER-devel-$LINUX_VER
+
+ARG CUDA_VER=11.2
+ARG PYTHON_VER=3.8
+ARG NUMPY_VER=1.21
+ARG RAPIDS_VER=21.08
+ARG UCX_PY_VER=0.21
+
+ADD https://raw.githubusercontent.com/dask-contrib/dask-sql/main/conda.txt /dask_sql_requirements.txt
+
+RUN conda config --set ssl_verify false
+
+RUN conda install -c gpuci gpuci-tools
+
+RUN gpuci_conda_retry install -c conda-forge mamba
+
+RUN gpuci_mamba_retry env create -n dask_sql --file /dask_sql_requirements.txt -c conda-forge
```

Comment:
This might be a bit of a problem if we are _also_ trying to respect the `PYTHON_VER` .  The conda env file doesn't define a python version

Reply:
Good point - we can probably do a hacky solution for now and just append the Python version to the local copy of the file, but I imagine long term we might want environment YAML files for various Python versions

Reply:
Opened a dask-contrib/dask-sql#238 for this; until that is merged in, I'm going to have gpuCI pull in the conda files from my branch and build the environment using those